src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,7 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1==bet1&&result2==bet2)?2:(result1>result2 && bet1>bet2 || result1<result2 && bet1<bet2|| result1 ==result2 && bet1==bet2)?1:0;
```

Comment:
Code Style:
-line length <= 100,
-whitespaces around variables and operands,
-empty line after return statement is redundant.
Please, view this article: [https://mate-academy.github.io/style-guides/java/java.html](url)